[eve-kernel-arm64-v6.1.38-generic] config: enable tracing like for amd64

christoph-zededa opened the pull request.

christoph-zededa committed:
config: enable tracing like for amd64

Signed-off-by: Christoph Ostarek <christoph@zededa.com>

rene commented on arch/arm64/configs/eve_defconfig:
```diff
@@ -1230,5 +1230,9 @@ CONFIG_SOFTLOCKUP_DETECTOR=y
+CONFIG_FUNCTION_PROFILER=y
+CONFIG_STACK_TRACER=y
+CONFIG_FTRACE_SYSCALLS=y
+CONFIG_BLK_DEV_IO_TRACE=y
```
@christoph-zededa , did you run `make savedefconfig`  after editing your changes? For me it also adds the CONFIG_RELAY:

```diff
--- build/defconfig	2024-07-18 09:30:13.270323120 +0000
+++ arch/arm64/configs/eve_defconfig	2024-07-18 09:25:15.632827928 +0000
@@ -41,6 +41,7 @@ CONFIG_CGROUP_BPF=y
 CONFIG_NAMESPACES=y
 CONFIG_USER_NS=y
 CONFIG_CHECKPOINT_RESTORE=y
+CONFIG_RELAY=y
 CONFIG_BLK_DEV_INITRD=y
 CONFIG_EXPERT=y
 CONFIG_SGETMASK_SYSCALL=y
```

christoph-zededa replied:
I added this change as a separate commit as it is also in `eve-kernel-arm64-v6.1.38`.

christoph-zededa committed:
config: synchronize with savedefconfig

make ARCH=arm64 eve_defconfig
make ARCH=arm64 savedefconfig
cp defconfig arch/arm64/configs/eve_defconfig

Signed-off-by: Christoph Ostarek <christoph@zededa.com>

rene reviewed (approved):
LGTM
